paddle/capi/CMakeLists.txt

```diff
@@ -55,7 +57,7 @@ endif()
 install(FILES ${CAPI_HEADERS} DESTINATION include/paddle)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/config.h DESTINATION include/paddle)
 if(ANDROID)
-  install(TARGETS paddle_capi_whole paddle_capi_shared
+  install(TARGETS paddle_nn_engine paddle_layers paddle_capi_shared
```

Comment:
Do we need to install `paddle_capi_whole` for a period of time?

Reply:
Done.